lib/ask_web/controllers/respondent_controller.ex

```diff
@@ -1117,8 +1127,7 @@ defmodule AskWeb.RespondentController do
                   ((e.respondent_hashed_number == ^last_hash and e.id > ^last_id) or
                      e.respondent_hashed_number > ^last_hash),
               order_by: [e.respondent_hashed_number, e.id],
-              limit: 1000,
-              preload: :channel
+              limit: 1000
```

Comment:
Another question, more for my own learning than for a review ... Could this batching size change something if we increase it, i.e., to 2000?

Reply:
It will load 2000 respondents in memory on each batch, meaning that it would halve the number of SQL requests, but double the number of respondents to return at once, and also double the memory usage. It's all about tradeoff and fine tuning. You can try it out, and see what's the impact.

I often see batches at 1000. It's neither too big (limited memory usage) nor too small (too many queries).